projects/openstreetmap-website-notes-refactoring/README.md

```diff
@@ -0,0 +1,38 @@
+# OpenStreetMap-Website: Refactor Notes
+
+The goal of this project is to refactor the existing notes implementation on the OSM website.
+
+* Project: [openstreetmap-website](https://github.com/openstreetmap/openstreetmap-website)
+* Related:
+  - https://github.com/openstreetmap/openstreetmap-website/issues/3831
+  - maybe https://github.com/openstreetmap/openstreetmap-website/pull/3934
+
+The expected deliverable is a mergeable pull request towards https://github.com/openstreetmap/openstreetmap-website. It must at least:
+- Follow the idea of a refactoring by changing as little as possible in behaviour and improving the underlying code structure
+- Handle the UI part – Map, Forms, Lists, …
+- Handle the API part – without any breaking changes to the API
+- Handle the Data migration of existing data – discuss details with the operations team
+- Handle freatures related to moderation and administration
```

Comment:
> freatures
Should be 'features'.

Reply:
Thanks @danieldegroot2. Btw, are you aware of this feature 
<img width="238" alt="image" src="https://github.com/osmfoundation/ewg_bidding/assets/111561/63281b95-0ee3-48a1-9445-1f00a36659a0">?

Reply:
I've mostly reviewed existing commits, so I haven't used it I suppose.